### ISSUE TITLE
Double file path for some manifest icons

### ISSUE THREAD

**Issue by adriaandotcom** [BlackGlory/parse-favicon#8]
For [this minifest](https://nodejs.org/static/images/favicons/manifest.json):

```json
{
  "name": "Node.js",
  "short_name": "Node.js",
  "icons": [
    {
      "src": "/static/images/favicons/android-chrome-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "/static/images/favicons/android-chrome-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "theme_color": "#333333",
  "background_color": "#333333",
  "display": "standalone"
}
```

It returns this:

```
icon {
  url: '/static/images/favicons/static/images/favicons/android-chrome-192x192.png',
  reference: 'manifest',
  type: 'image/png',
  size: { width: 192, height: 192 }
}
```

The `url` shouldn't have double `static/images/favicons` in them.

**Comment by BlackGlory** [comment]
v3.0.10 should fix this bug.

In order to make the interface clearer, I replaced `undefined` in the interface with `null`, and released v4.0.

**Comment by adriaandotcom** [comment]
You're so fast, it's amazing. Thanks again!
